crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1003,9 +1003,12 @@ spec:
                     items:
                       type: string
                       enum: ["logs-tab"]
+                  istio_annotation_action:
+                    description: "Flag to enable/disable an Action to edit annotations."
+                    type: boolean     
                   istio_injection_action:
                     description: "Flag to enable/disable an Action to label a namespace for automatic Istio Sidecar injection."
-                    type: boolean
+                    type: boolean                   
```

Comment:
trivial - does this line have a bunch of end-of-line spaces? did something else change here?